app/views/admin/shared/_tabnav_product.html.haml

```diff
@@ -1,23 +1,22 @@
-- product = @instrument || @service || @item || @bundle || @product
-- return nil if product.nil? || product.id.nil?
+- return nil if @product.nil? || @product.id.nil?
```

Comment:
I'm honestly not quite sure what this check does, but as this was a first-step refactor, I left it as-is.